autopr/ai_service.py

```diff
@@ -161,7 +162,140 @@ def get_pr_description_suggestion(commit_messages: list[str]) -> tuple[str, str]
             return "[AI returned empty response]", ""
```

Comment:
Consider handling the case where `client` is not initialized before this function is called, to prevent `NameError` or `AttributeError`. You could either initialize `client` within this function if it's not already, or raise a more descriptive error if `client` is expected to be initialized elsewhere.